src/app/modules/main/activity_center/module.nim

```diff
@@ -303,23 +303,5 @@ method setActivityCenterReadType*(self: Module, readType: int) =
 method getActivityCenterReadType*(self: Module): int =
   return self.controller.getActivityCenterReadType().int
 
-method groupCountersChanged*(self: Module) =
-  self.view.groupCountersChanged()
-
-method getAdminCount*(self: Module): int =
```

Comment:
You removed a bunch of methods here but they are still in io_interface

Reply:
Oh, i expected it won't compile with unimplemented methods. Fixed :)